### DISCUSSION TITLE
question for company usage

### DISCUSSION THREAD

**Discussion by IzunaMoon** [rustdesk/rustdesk-server-pro#82] [General]
edited due extremely bad english

my first question is can clients connect to echt other without a user/license if i upgrade to rustdesk server pro
(for axample this user only need to connect to only 1 device so doesnt need de extra options dat comes with pro)

a second question will older clients like version 1.1.9 still work i have check hour devices a i have over 200 devices to update so i rather ask
i know we best upgrade them all but it be great if they remain working just in case

**Comment by dinger1986** [comment]
Can you write this again but in English?

**Reply by IzunaMoon** [reply]
right sorry

my first question is can clients connect to echt other without a user/license if i upgrade to rustdesk server pro
(for axample this user only need to connect to only 1 device so doesnt need de extra options dat comes with pro)

a second question will older clients like version 1.1.9 still work i have check hour devices a i have over 200 devices to update so i rather ask
i know we best upgrade them all but it be great if they remain working just in case

ps; sorry for my poor english not my main languege

**Reply by dinger1986** [reply]
Hello,

Thanks for that, sorry it was the replacing th for d what was confusing me.

Controlling devices need to be signed in so you would need a user.

The old clients should still be able to be connected to however you can't sign in etc so should really upgrade them.
